pkg/modelconfig/configmap.go

```diff
@@ -13,8 +13,8 @@ var logger = log.Log.WithName("ModelConfig")
 var json = jsoniter.ConfigCompatibleWithStandardLibrary
 
```

Comment:
Since Yao's PR is merged you can remove this file and use https://github.com/kubeflow/kfserving/tree/master/pkg/modelconfig

Reply:
I am, as per https://github.com/kubeflow/kfserving/pull/989/files#diff-5b2482c98bf5c61625b6fb19a69b7477R87, these changes are just after running `make fmt vet`